Resources/Prototypes/Entities/Structures/Storage/Canisters/gas_canisters.yml

```
@@ -111,6 +111,10 @@
     - type: GuideHelp
       guides:
       - GasCanisters
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)